source/help/settings/channel-settings.md

```diff
@@ -12,7 +12,10 @@ By default, the desktop notification preference assigned in *Account Settings* i
 Customize what activity causes the channel name to be bolded by accessing the channel name drop-down menu, then click **Notification Preferences > Mark Channel Unread**.
 
 ## Channel Header
-The header is the text that appears next to the channel name at the top of the screen. It can be used to summarize the channel topic or provide links to frequently accessed documents. Any channel member can set the channel header by clicking the channel name at the top of the center pane to access the drop-down menu, then click  **Set Channel Header**.
+Select **Edit Channel Header** to change the text that appears next to the channel name at the top of the screen. It can be used to summarize the channel topic or provide links to frequently accessed documents. Any channel member can edit this setting, unless the System Administrator has [restricted the permissions](https://docs.mattermost.com/administration/config-settings.html#enable-public-channel-management-permissions-for).
 
 ## Channel Purpose
-This text appears in the channel list in the *More…* menu and helps others decide whether to join. Any channel member can set the channel purpose by clicking the channel name at the top of the center pane to access the drop-down menu, then click  **Set Channel Purpose**.
+Select **Edit Channel Purpose** to change the text that appears in the *More…* menu for channels. The channel purpose is usually a short description that helps others decide whether to join the channel. Any channel member can edit this setting, unless the System Administrator has [restricted the permissions](https://docs.mattermost.com/administration/config-settings.html#enable-public-channel-management-permissions-for).
+
+## Rename Channel
```

Comment:
Propose naming this section "Channel Name" to be consistent with other sections?